src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -0,0 +1,204 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_ADDRESS;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.function.Predicate;
+
+import org.junit.jupiter.api.Test;
+
+import javafx.collections.ObservableList;
+import seedu.address.commons.core.GuiSettings;
+import seedu.address.logic.commands.SortCommand.SortArgument;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.Prefix;
+import seedu.address.model.Model;
+import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.ReadOnlyUserPrefs;
+import seedu.address.model.person.Person;
+import seedu.address.model.tag.Tag;
+
+class SortCommandTest {
+
+    @Test
+    public void constructor_nullPerson_throwsNullPointerException() {
```

Comment:
Hi, I encountered this while looking for inspirations to write my search test. Is the method name `nullPerson` intended?

Reply:
Hmm... perhaps I should change it to `nullList` instead. Thanks for pointing it out!